src/components/SingleColCard/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Row, Col, Image, Button } from "antd";
+import { NavLink } from "react-router-dom";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+const SingleColCard = ({ post, order, isTesting }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Row className="justify-content-md-center articleContainer2">
+        <div className="containerBlog2">
+          <Col lg={{ span: 3, order: order }} className="articleImage">
+            <Image
+              src={post.jetpack_featured_media_url}
+              className="roundImage2"
+              roundedCircle
+            />
+          </Col>
+          <Col lg="6" className="blogCardBox2">
+            <h3
+              className="articleTitle2"
+              dangerouslySetInnerHTML={{
```

Comment:
Is there any other way to do this, rather than innerHTML? What do you think @halitbatur ?

Reply:
@halitbatur do you know another way?

Reply:
@bertacsevercan @gizemcdeniz I think innerHTML needs to be used here no other way